### PR TITLE
Feat toggle content show

### LINKED ISSUE #7005: No option to hide nsfw content after you click show
This is a followup for #5772, or at least for the gallery item, on explore is still is not blurred :D 
image
- lets change the show button, no hover effect, just simple button with background
- when you click show, button changes to hide content

you can test here: https://canary.kodadot.xyz/rmrk/gallery/18883932-9A78FF2AECF9BED171-DOTINSIDER-POLKADOTINSIDER-0000000000000001

updated in figma handoff: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=1287%3A18959&mode=design&t=2KWtprPqfstWkfBj-1

### PR DESCRIPTION
**Pull request by xiiiAtCn**
## PR Type

- [ ] Bugfix
- [x] Feature
- [x] Refactoring

## Context

- [x] Closes #7005 
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] my DOT address: [Payout](https://kodadot.xyz/dot/transfer?target=12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW&usdamount=8.82&donation=true)

#### Community participation

- [x] [Are you at KodaDot Ecosystem Telegram?](https://t.me/kodadot_eco)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### <samp>🤖 Generated by Copilot at 1f1c472</samp>

Refactor `MediaItem.vue` to use Vue 3 script setup and `NeoButton` component, and add translation key for `hideContent` label. These changes improve the user experience and the code quality of the media item component.

### <samp>🤖 Generated by Copilot at 1f1c472</samp>

> _`MediaItem` shines_
> _With script setup and button_
> _Autumn of refactor_

### CODE REVIEW

File: libs/ui/src/components/MediaItem/MediaItem.vue [hunk 1]
```diff
@@ -11,138 +11,137 @@
...
     <div
       v-if="isLewd && isLewdBlurredLayer"
-      class="nsfw-blur is-flex is-align-items-center is-justify-content-center is-flex-direction-column">
+      class="nsfw-blur is-capitalized is-flex is-align-items-center is-justify-content-center is-flex-direction-column">
       <NeoIcon icon="eye-slash" class="mb-3" />
-      <span class="heading-nsfw"> {{ $t('lewd.explicit') }} </span>
-      <span class="nsfw-desc">{{ $t('lewd.explicitDesc') }}</span>
-      <span class="nsfw-content" @click="showContent()">{{
-        $t('lewd.showContent')
+      <span class="nsfw-heading has-text-weight-bold">
```
Comment (daiagi) [inline libs/ui/src/components/MediaItem/MediaItem.vue:16]: `nsfw-heading` class is not used

File: libs/ui/src/components/MediaItem/MediaItem.vue [hunk 2]
```diff
@@ -11,138 +11,137 @@
...
-    text-align: center;
+    .nsfw-desc {
+      max-width: 18.75rem;
+    }
   }
-
-  .nsfw-content {
-    cursor: pointer;
-    bottom: 3.125rem;
+  .nsfw-action {
```
Comment (daiagi) [inline libs/ui/src/components/MediaItem/MediaItem.vue:129]: better to rename this class to `no-border`

check if global.scss have this kind of utility classes already
if it doesn't , move it there
and if it does, use it

File: libs/ui/src/components/MediaItem/MediaItem.vue [hunk 3]
```diff
@@ -11,138 +11,137 @@
...
+      <span class="nsfw-desc text-align-center">{{
+        $t('lewd.explicitDesc')
       }}</span>
     </div>
+    <NeoButton
+      v-if="isLewd"
+      rounded
+      no-shadow
+      class="nsfw-action px-4 py-1 is-size-6"
+      :class="[isLewdBlurredLayer ? '' : 'hide']"
```
Comment (daiagi) [inline libs/ui/src/components/MediaItem/MediaItem.vue:28]:
```suggestion
:class="{'hide': isLewdBlurredLayer }"
```

**Review by daiagi — CHANGES_REQUESTED** [review]
last round of changes from me

**Comment by xiiiAtCn** [comment]
> last round of changes from me

@daiagi  very appreciate for spending lots of time to review my code and give me useful advices

**Review by daiagi — APPROVED** [review]

**Comment by daiagi** [comment]
@prury please test once again

**Comment by prury** [comment]
looks good on dark/white mode, desktop and mobile ✔️

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 4.46 USD/DOT ~ 11.211 $DOT
🧗 [12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW](https://polkadot.subscan.io/account/12jXHf7yYF4wTRjDPbwFB24V1nVRPRkuU2S8Ke5ZWTfvyBGW)
🔗 [0x351b807993fc80bdf91da300bb37da09d6a6daf0b13c49d506dc4ae9df541de5](https://polkadot.subscan.io/extrinsic/0x351b807993fc80bdf91da300bb37da09d6a6daf0b13c49d506dc4ae9df541de5)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**